Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/Merchant/MerchantMessage.hs

```diff
@@ -25,6 +25,10 @@ data MessageKey
   | WELCOME_TO_PLATFORM
   | ALTERNATE_NUMBER_OTP
   | ONBOARD_SUPPORT_SMS_TEMPLATE
+  | SEND_END_RIDE_MESSAGE
+  | SEND_ONBOARDING_YATRI_MESSAGE
+  | SEND_BOOKING_MESSAGE
+  | SEND_CASH_COLLECTED_MESSAGE
   deriving (Generic, Show, Read, FromJSON, ToJSON, Eq, Ord)
```

Comment:
Lets remove `SEND` prefix from all of these as its redundant.

Reply:
removed SEND prefix